apps/aehttp/src/aehttp_app.erl

```diff
@@ -69,31 +74,52 @@ start_swagger_internal() ->
 start_websocket_internal() ->
     Port = get_internal_websockets_port(),
     PoolSize = get_internal_websockets_acceptors(),
+    ListenAddress = get_internal_websockets_listen_address(),
     Dispatch = cowboy_router:compile([
         {'_', [
             {"/websocket", ws_handler, []}
         ]}
     ]),
     {ok, _} = cowboy:start_http(http, PoolSize,
-                                 [{port, Port}, {ip, {127, 0, 0, 1}}],
+                                 [{port, Port}, {ip, ListenAddress}],
                                  [{env, [{dispatch, Dispatch}]}]),
     ok.
 
+get_and_parse_ip_address_from_config_or_env(CfgKey, App, EnvKey, Default) ->
+    Config = aeu_env:user_config_or_env(CfgKey, App, EnvKey, Default),
+    case is_binary(Config) of
```

Comment:
Why do you need this check, in what case the config value is not binary?

Reply:
I set the `-define` statements to be tuples again, like the defaults really were before.

Reply:
I would prefer binary defaults to get rid the of this extra checks